index.css

```diff
@@ -832,41 +890,50 @@ body {
   }
 
   .imageleft {
-    float: left;
-    width: 50%;
     padding: 5px;
+    position: relative;
+    width: 50%;
+    float: left;
   }
 
-  .imageleft img {
+  .photo {
+    display: block;
     width: 100%;
+    height: auto;
   }
 
   .overlay {
     position: absolute;
-    top: 0;
     bottom: 0;
-    left: 0;
+    left: 100%;
     right: 0;
+    background-color: #ec5242;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 👍 Well done!

Reply:
Please explain further. I have not understood your request

Reply:
Happy to do so but on which request? 🧐

Reply:
Please just explain here so that i can make the change if need be

Reply:
I don't understand which request you are referring to. You commented on a comment that I was congratulating your use of colors 🤓 Can you please tell me what request you need help understanding?